app-authz-rest-springboot/src/main/java/org/keycloak/quickstart/springboot/config/KeycloakSpringBootResolverConfig.java

```diff
@@ -0,0 +1,15 @@
+package org.keycloak.quickstart.springboot.config;
```

Comment:
Requiring KeycloakSpringBootResolverConfig is actually a regression in the Spring Boot adapter. Making changes like these to QuickStarts are simply masking breaking compatibility. See https://issues.redhat.com/browse/KEYCLOAK-13235 for example.

Reply:
What do you suggest than?